### DISCUSSION TITLE
High-level Architectural Blueprint

### DISCUSSION THREAD

**Discussion by dasunhegoda** [spdci/.github#12] [Ideas]
Hi Everyone,

I propose creating a high-level architectural blueprint that outlines the main components and interactions within our ecosystem. This would offer a broad overview (30,000 ft. view) for stakeholders, highlighting the various players and the overall scope of the ecosystem. The high-level architectural blueprint provides a clear and concise overview, ensuring all stakeholders understand the system's core components and their interrelationships.

What are your thoughts on this approach?

Thanks

**Comment by krisaccessh** [comment]
Hi Dasun and everyone,

That's indeed a very good suggestion. I suggest to begin a focused group discussion thread on architectural blueprint topic to identify the broader contours of the architectural blueprint. As DCI team has initially covered interoperability between SP systems and CRVS in  first workstream discussion and lot of work has already been done by Mosip and G2pconnect groups in ID and payment streams, we need to align collaborate thoughts from all groups and develop a consensual approach to define a comprehensive architectural blueprint . I personally propose to use X-road as a standard data exchange layer to connect the various components in the architectural blueprint Please suggest if you have some other thoughts on this. @vvujjini @dasunhegoda @jeremi  please suggest your thoughts and when we can be available for this focused group discussion

**Reply by saurav-bhattarai** [reply]
Would you be looking at something similar to OpenHIE from the health space ? https://guides.ohie.org/arch-spec/architecture-specification/overview-of-the-architecture

I'm sure @uwahser would like to take part in these discussions :)

**Reply by dasunhegoda** [reply]
Thank you for the insight.

While we can draw inspiration from X-Road, it's essential that our blueprint remains neutral and doesn't favor any specific product or project. OpenHIE could also serve as a valuable reference point as we proceed, even though it's health-driven.

**Reply by krisaccessh** [reply]
Thanks Saurav and Dasun
A consensus driven approach to define architectural blueprint will be helpful to make a neutral but comprehensive architectural blueprint. I request all tech members  to  be part of this discussion and provide your valuable insights
